includes/classes/Product.php

```diff
@@ -244,6 +244,10 @@ protected function loadProductDetails(int $product_id, ?int $language_id = null)
         $data['linked_categories_count'] = $results->RecordCount();
         $data['linked_categories'] = [];
         foreach($results as $result) {
+            if ($result['categories_id'] === $data['master_categories_id']) {
+                $data['linked_categories_count'] --;
```

Comment:
No space between a unary operator and its operand.

Reply:
Space removed before the `--`